readme.md

```diff
@@ -164,17 +164,3 @@ Visit [issues](https://github.com/tabasumu/jambo/issues) to get started.
 ## Credits
 [Timber](https://github.com/JakeWharton/timber)
 
-## License
```

Comment:
@ArunKumar-07 please remove the tag that creates  a badge not the license itself. There is a license badge